### PR TITLE
Von branch

### PR DESCRIPTION
**Pull request by nogadas-vonryan**
Added comment functionality with ascend/descend sorting - [nogadas_vonryan/index.js](https://github.com/PUP-BSIT/exercise-13-tuf_developers/compare/main...Von-branch#diff-517a19f7eb72eed3ac497f716b2cfb95771cb4a58c23b1ccc29b0a22fdc9398f)
Added netlify links - [README.md](https://github.com/PUP-BSIT/exercise-13-tuf_developers/compare/main...Von-branch#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

### CODE REVIEW

File: nogadas_vonryan/index.html [hunk 1]
```diff
@@ -108,34 +108,31 @@ <h2>My Goals</h2>
...
 				<h2>Comment Section</h2>
-				<ul>
-					<li>Your goals are very ambitious! -Jhef</li>
-					<li>I hope you achieve all your goals that will 
-						help you be successful in life - Andrea</li>
-					<li>Keep on going hard work pays off - Mau</li>
-					<li>Your perspective is refreshing - Mark</li>
-				</ul>
+				<select id="sort_type" onchange="sortComments()">
+					<option value="newest_first">Newest First</option>
```
Comment (Valerio-SirLance) [inline nogadas_vonryan/index.html:112]: by checking your Netlify link, I notice that the "Newest First" option displays the oldest comments, while the "Oldest First" displays the new ones.

**Review by Valerio-SirLance — CHANGES_REQUESTED** [review]

File: nogadas_vonryan/index.html (hunk 1 above)
Comment (nogadas-vonryan) [reply nogadas_vonryan/index.html:112]: @SLValerio 
```fix inverted sorting function ``` - cda39b93b6c9b7f57778070850e666339ae5dbca

Netlify link was updated. Thank you po!

**Review by Valerio-SirLance — APPROVED** [review]

**Comment by Valerio-SirLance** [comment]
great! you can now merge this to main branch and close this pull request.

**Comment by nogadas-vonryan** [comment]
Thank you po! Merging now.

**Pull request merged**